Icon yw4z Calibration Tab + fix

KrisMorr opened the pull request:
@yw4z
Firstly, thank you for your work! It's great! After switching to the BambuLab printer, a new tab for calibration appears. It has an icon with the same name as Devices. Personally, I don't like it. This PR includes that fix, which allows changing this icon.


![tab](https://github.com/yw4z/OrcaSlicer/assets/154343071/69b09763-3bb2-410a-ad11-3d1d5609a77c)

bistory committed:
Updated French translations (#4427)

* New translations

* Changed "écharpe" by "biseau"

* Fixed "tour de nettoyage" by "tour de purge"

KrisMorr committed:
Polish translation update (#4430)

* Polish translation update

* Polish translation update

tadeu2 committed:
Update Spanish translation (#4445)

Update Spanish translation

PhenixNoir committed:
Update Turkish Language translation (#4454)

SoftFever committed:
update corresponding locale files

cochcoder committed:
Minor spelling & grammatical fixes (#4444)

* Minor spelling & grammatical fixes

* Correct counterbole to counterbore

---------

Co-authored-by: SoftFever <softfeverever@gmail.com>

nevack committed:
chore(slicer): Update copyright year to 2024 (#4428)

Also remove duplicated definition for `MACOSX_BUNDLE_COPYRIGHT`.

Signed-off-by: Dzmitry Neviadomski <nevack.d@gmail.com>

SoftFever committed:
Handle parameter name change properly.
Fix build break and keep compatibility to existing user preset after #4444 changes

SoftFever committed:
Fix alternate extra wall warning: change ensure vertical shell thickness to Moderate instead of None (#4463)

* fix warning

* Update locale

KrisMorr committed:
Changing the background color of the thumbnail (#4421)

rodriguezst committed:
Allow AppImage generation in aarch64 build hosts (#4410)

Get arch from build host

It was harcoded to x86_64 and was failing to generate AppImage files on aarch64 build hosts

SoftFever committed:
Fixed an issue that in some cases, the OrcaSlicer_profile_validator didn't report errors properly

bistory committed:
Further FR translation fixes (#4472)

* New translations

* Changed "écharpe" by "biseau"

* Fixed "tour de nettoyage" by "tour de purge"

* Changed translation of "solid" to "plein(e)"

* Fixed Printing by object G-code

* Make "wipe" and "prime" translations consistent

* More coherent translation for prime tower

hliebscher committed:
fix de new feature (#4479)

KrisMorr committed:
Merge pull request #49 from yw4z/icon-designs

Icon designs

KrisMorr committed:
tab_calib_sf_update

KrisMorr committed:
tab_calib_sf_update

KrisMorr committed:
fix MainFrame

yw4z commented:
Thanks i will add a file with tab_calib_sf.svg to my project. also i have a list in main [commit](https://github.com/SoftFever/OrcaSlicer/pull/4368) adding new icons in UI and there are many bugs / inconsistencies in that. i have not much enough experience at coding any help on these appreciated

KrisMorr committed:
fix MainFrame

GlauTechCo committed:
Update OrcaSlicer_tr.po (#4490)

SoftFever committed:
Fixed an issue that changing printer selection clears out Process overrides (#4506)

SoftFever committed:
Add bed types WIKI document and link

SoftFever committed:
update adaptive bed mesh wiki for Marlin

crwusiz committed:
kor translation update (#4503)

SoftFever committed:
Fix an issue that only 1 bottom/top will be generated when ensure vertical thickness is set to None (#4504)

SoftFever committed:
fix build break error for multiconfig generator on Mac

SoftFever committed:
Handle travel acceleration warning properly for different firmwares

SoftFever committed:
disable speed check

SoftFever committed:
Add safety check for seam_slope_start_height

SoftFever committed:
update locale and Chinese translations

SoftFever committed:
fix a crash issue when check machine limits

SoftFever committed:
set default logging level to warning and make it configurable in .conf file

gringer committed:
3D Honeycomb - switch direction at smallest bridge point, rather than every layer (#4425)

Co-authored-by: SoftFever <softfeverever@gmail.com>

SoftFever committed:
Minor tweak: avoid exceptions

SoftFever committed:
Properly handle "dev" version

SoftFever committed:
Prevent crashes from edge cases when slicing complex small model with "big" fuzzy skin is used

AndylgTom committed:
Russian translation update (#4528)

Russian translation update OrcaSlicer V2.0.0 beta

SoftFever committed:
log tweak:
1. deprecate severity_level, use log_severity_level config key
2. remove encrytped bbl debug_network log file

SoftFever committed:
restore tools to git from lfs

ohmdelta committed:
Add Docker Build Support (#4485)

init docker

hliebscher committed:
Update German translations (#4529)

2  fixes

KrisMorr committed:
Polish translation update (#4541)

* Update Polish translation

* Update Polish translation

* Polish translation update

davidjuanesb committed:
Catalan language updated to OrcaSlicer-v2.0.0-beta (#4540)

bistory committed:
Ender 3 V3 : Some improvements for supports and machines limits (#4527)

* Some improvements for supports and machines limits

* Increase z-hop for KE to avoid the nozzle hit the parts

* Changed jerk V3 KE settings to more logical values.

* Decreased jerk

* Speed, acceleration and jerk settings fine-tunings.

Hukete committed:
Update QIDI profiles (#4549)

1）add Q1 Pro profiles

yw4z committed:
Minified SVG Bamboo Lab Printer images (#4513)

Used this site https://svgomg.net
Minifier basically joins lines / curves with same style to reduce size
Doesn't change geometry
Size dropped to 2,02MB from 4,75MB
Couldn't tested on UI. Not sure how to do that

igiannakas committed:
Inner Outer Inner mode - arachne bug fix (#4416)

* Inner Outer Inner mode - arachne bug fix

* Update PerimeterGenerator.cpp

igiannakas committed:
Revised Inner Outer Inner mode for Classic mode (#4417)

cochcoder committed:
Add Ender-3 Pro (#4573)

* Add Ender-3 Pro cover

* Add Ender-3 Pro

* Add Ender-3 Pro .2, .4, .6, and .8

* Add Ender-3 Pro processes

* Delete Ender-3 Pro_cover.png

* Add a smaller version of Ender-3 Pro_cover.png

* Delete Ender-3 Pro_cover

* Further image improvements

frankieorabona committed:
Further IT translation fixes (#4569)

* Update OrcaSlicer_it.po

SoftFever committed:
Fix crashes caused by self destructed splash screen.

SoftFever committed:
Fix wrong filaments appreared in other manufacture

FlashforgeOfficial committed:
Update Flashforge Printer Profiles for Guider3 Ultra 0320 (#4587)

Update Printer Profiles for Guider3 Ultra 0320

bistory committed:
Fix for missing K1 references (#4600)

Fix for fix wrong filaments appreared in other manufacture

cochcoder committed:
Fix Anker M5C 0.6 bed size (#4608)

iherbak committed:
Falashforge 5M network support (#4546)

* build action changes

* Adding flashforge 5M series network support

* Update CMakeLists.txt

* Update Flashforge.cpp

* reverting files for pull request

---------

Co-authored-by: SoftFever <softfeverever@gmail.com>

yw4z committed:
Optimized plate models of CONSTRUCT3D (#4567)

Size reductions
construct_1_xl_buildplate_model > 1073 from 2250KB
construct_1_buildplate_model > 599 from 608KB
Just joined parts and optimized planar surfaces. didn't changed borders

Also adding a feature for creating 3D bed models from SVG graphics will give a nice reduction on size. Constantly new printers added to profiles. it will be nice upgrade to reduce size of folder

KrisMorr committed:
Added a string 'no model information' for translation (#4623)

* Add files via upload

* Added a string for 'no model information'

cochcoder committed:
Fix Ender-3 Pro missing from Generic Creality filaments (#4630)

Fix Ender-3 Pro missing from generic filaments

FlyingbearOfficial committed:
fix start_code and 0.16mm parameters (#4631)

* Update FlyingBear S1 0.4 nozzle.json

* Update 0.16mm Optimal @FlyingBear Reborn3.json

* Update 0.16mm Optimal @FlyingBear S1.json

* Update fdm_process_common.json

* Update fdm_klipper_common.json

* Update fdm_machine_common.json

* Update fdm_klipper_common.json

* Update fdm_machine_common.json

SoftFever committed:
Feature/home page improve (#4602)

* Add new MyRRF custom printer

* hide user panel when networking plugin is not installed/used

* Fixed a bug that promption of installing plugin notification was not displayed when trying to switch to Bambu device tab

* replay missing_connection animated image when switching to device tab

bistory committed:
Raised bed temperatures for Ender 3 V3 PLA profile (#4636)

bistory committed:
Enable detect_thin_wall for Qidi printers (#4637)

cochcoder committed:
Fix missing Construct3D covers (#4632)

* Rename CONSTRUCT 1 XL_cover.png to Construct 1 XL_cover.png

* Rename CONSTRUCT 1_cover.png to Construct 1_cover.png

SoftFever committed:
update locales

SoftFever committed:
bump version to 2.0-rc

Noisyfox committed:
Support for SimplyPrint cloud integration (#4525)

* Make httpserver more generic and reusable

* Add OAuthJob

* Fix issue caused by the fact that the backing widget of the `TextCtrl` is no longer `wxTextCtrl`

* Implement login and token refresh

* Implement file upload

* Try fix build error

* Support BBL printers

* Show error message if user hasn't done OAuth

* Fix typo

* Update error message

* Disable unsupported options when SimplyPrint is selected

igiannakas committed:
Revert automatic application of reduced internal bridge filtering for lightning infill (#4639)

hliebscher committed:
Update German Translations (#4640)

Noisyfox committed:
Fix issue that painter sphere is rendered at the wrong position (#4652)

SoftFever committed:
Wrong bridge detection result (#4651)

process_external_surfaces before discover_horizontal_shells

SoftFever committed:
update locale

bistory committed:
Updated French translations (#4654)

SoftFever committed:
fix "G92 E0" warning for Guider 3 ultra and Rase3D Pro

SoftFever committed:
bump profile version to 2.0.1

KrisMorr committed:
Merge branch 'SoftFever:main' into icon_yw4z

KrisMorr committed:
upd_build

KrisMorr committed:
Update Polish translation (#4658)

SoftFever committed:
Revert "Wrong bridge detection result (#4651)"

This reverts commit f626e4dd27692f892beccbbfe714cfa84730bc60.

KrisMorr committed:
Merge branch 'SoftFever:main' into icon_yw4z
